src/main/kotlin/no/skatteetaten/aurora/boober/utils/jsonNodeUtils.kt

```diff
@@ -36,11 +36,17 @@ val JsonNode.openshiftKind: String
 val JsonNode.openshiftName: String
     get() = if (this.openshiftKind == "deploymentrequest") {
         this.get("name")?.asText() ?: throw IllegalArgumentException("name not specified for resource kind=${this.openshiftKind}")
+    } else if (this.openshiftKind == "namespace") {
+        ""
     } else {
         this.get("metadata")?.get("name")?.asText() ?: throw IllegalArgumentException("name not specified for resource kind=${this.openshiftKind}")
     }
 
-
+fun JsonNode.changed(source: JsonNode, path: String): Boolean {
```

Comment:
This method has been named from the context of the caller (why are we calling it?) instead of by its function (what does the function do?). Since the function is quite generic, consider naming by from what it actually does; it does not check if something has changed, but checks if the values of a json path in two json documents/nodes are unequal.
For instance; notEqual
Also, the name of the JsonNode parameter should better reflect what it is. It is not a source, but the "other document/node"; notEqual(other: JsonNode, path: String)